js-core/homeworks/src/main.js

```diff
@@ -1,4 +1,147 @@
+/*
+Напишите функцию которая будет возвращать объект
+с свойством name, а значением будет первый аргумент функции
+*/
+function returnObject(name) {
+	let obj = {};
+	obj.name = name;
+	return obj;
 
-console.log(23);
+}
+var someObject = returnObject('Oleg');
+console.log(someObject);
 
+
+
+/*
+Функция будет принимать 1 аргумент - ОБЪЕКТ у которого
+будет свойство name
+и возвращать новый объект изменяя свойство name
+в большой регистр
+*/
+function toUpperCase(obj) {
+	obj.name = obj.name.toUpperCase();
+	return obj;
+}
+const objectWithName = { name: 'privet kak dela' };
```

Comment:
There some interesting moment, you return the same object but not a new.

So solution should be improved